wrapper/minecraft/src/main/kotlin/org/quiltmc/qkl/wrapper/minecraft/math/Vec2fOps.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qkl.wrapper.minecraft.math
+
+import net.minecraft.util.math.Vec2f
+import net.minecraft.client.util.math.Vector2f
+
+//region Standard math operators
+/**
+ * Adds a [Vec2f] to a [Vec2f].
+ */
+public operator fun Vec2f.plus(other: Vec2f): Vec2f {
+    return Vec2f(
+        this.x + other.x, 
+        this.y + other.y
+    )
+}
+
+/**
+ * Adds a Float to a [Vec2f].
+ */
+public operator fun Vec2f.plus(other: Float): Vec2f {
```

Comment:
is adding vectors and scalars together even defined?

Reply:
Another thing copied over from minecraft /shrug

Vec2f implements an `add(Float)` method like that, so I extended it to other vectors

If it doesnt make sense to keep it I could drop it though, its only for `Vec2f`, everything else is either immutable or only allows it component wise (i.e `add(Float, Float)`

Reply:
~~ugh, why Mojang, don't you understand how vector arithmetic works~~

yeah I'm in favor of dropping it, it just doesn't really make sense

Reply:
Alright sounds good